integrations/jira/jira-servicedesk-incidents/README.md

```diff
@@ -0,0 +1,91 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The Jira Service Management integration is a Sensu Event Pipeline that generates alerts in the Jira Service Management service.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration provides the following resources:
+
+* `jira-servicedesk-pipeline` [pipeline]
+* `jira-servicedesk-workflow` [workflow]
```

Comment:
the [workflow] link isn't defined at the bottom. I haven't been including workflow resources, though maybe I should?

Reply:
Correct; we don't list workflows here because they're not a first-class API resource.